### DISCUSSION TITLE
Custom Emoji are not displayed

### DISCUSSION THREAD

**Discussion by anLizard** [jagrosh/MusicBot#865] [Troubleshooting] [answered]
### Bug Description

Instead of the emoji itself, the bot's messages display the `:name:` of the emoji:

![image](https://user-images.githubusercontent.com/48686356/133906906-07bf9463-52f7-4d1b-be67-42f24219bc0c.png)

### Steps to Reproduce

1. Set emoji for success message, e.g. `success = "<:Fingerbang:888872320775618642>"`
2. Start the bot
3. Play any song and observe the message sent by the bot

### Expected Result

Bot's success message should show the custom emoji

### Debug Output

```shell
System Properties:
  java.version = 11.0.12
  java.vm.name = OpenJDK 64-Bit Server VM
  java.vm.specification.version = 11
  java.runtime.name = OpenJDK Runtime Environment
  java.runtime.version = 11.0.12+7
  java.specification.version = 11
  os.arch = amd64
  os.name = Linux

JMusicBot Information:
  Version = 0.3.5
  Owner = removed
  Prefix = -
  AltPrefix = null
  MaxSeconds = 3600
  NPImages = false
  SongInStatus = true
  StayInChannel = false
  UseEval = false
  UpdateAlerts = true

Dependency Information:
  JDA Version = 4.3.0_312
  JDA-Utilities Version = 3.0.5
  Lavaplayer Version = 1.3.78

Runtime Information:
  Total Memory = 162
  Used Memory = 25

Discord Information:
  ID = removed
  Guilds = 1
  Users = 6
```
```


### Additional Info

Emoji syntax is taken from https://github.com/jagrosh/MusicBot/issues/328#issuecomment-558942211

### Checklist

- [X] I have looked for information about this [within the documentation](https://jmusicbot.com/)
- [X] I have searched for similar issues [on the issues page](https://github.com/jagrosh/MusicBot/issues?q=is%3Aissue)
- [X] I am running the latest version of the bot: ![Release](https://img.shields.io/github/release/jagrosh/MusicBot.svg)

**Comment by MichailiK** [comment, marked as answer]
The bot must be in the server the emoji originates from, as well as having Use External Emojis permission (if the emoji comes from a different server).

**Reply by anLizard** [reply]
The bot does have the permission and is only used on the server that has the emojis on it

**Reply by MichailiK** [reply]
Your emoji doesn't exist then. I have [tried retrieving the emoji](https://cdn.discordapp.com/emojis/888872320775618642.png) using the emoji id and Discord responded with a 404. Copy & paste the emoji id again.

**Comment by anLizard** [comment]
Managed to fix it... I used the wrong ID for the emotes.

For furute readers: Open the emote in the browser, copy the file name.
My mistake was copying the messageID instead.
